src/plugins/workspace/public/plugin.ts

```diff
@@ -407,9 +415,30 @@ export class WorkspacePlugin implements Plugin<{}, {}, WorkspacePluginSetupDeps>
       });
     }
 
+    // register workspace list in home page
+    this.registerWorkspaceListToHome(core, contentManagement);
+
     return {};
   }
 
+  private registerWorkspaceListToHome(
+    core: CoreStart,
+    contentManagement: ContentManagementPluginStart
+  ) {
+    if (contentManagement) {
+      contentManagement.registerContentProvider({
+        id: HOME_PAGE_ID,
```

Comment:
I guess the id is used for the content, not the page.

Reply:
Yes, this is the content provider id, you can give it any unique string, like `workspace_list_card`

Reply:
thanks for catching this, update to `workspace_list_card_home`